string/strerror.c

```diff
@@ -68,7 +69,88 @@ static inline const char *strerror_(const errinfo_t *tab, size_t tabsz, const ch
 
 char *strerror(int errnum)
 {
-	return (char *)strerror_(errtab, sizeof(errtab), errnames, errnum);
+	switch (errnum) {
+		// Error labels according gnu
+		case EPERM: return "Operation not permitted";
+		case ENOENT: return "No such file or directory";
```

Comment:
we could probably align `return`'s to make them more readable.

To silent the `clang-format` You would need to surround the code block with `/* clang-format off */` and `/* clang-format on */`